### DISCUSSION TITLE
Why is radiogroup always required?

### DISCUSSION THREAD

**Discussion by Stupidism** [payloadcms/payload#1257] [Q&A]
I have a radio group field that's marked as not required, but it still prompts error when I save without checking any radio button.

<img width="1023" alt="image" src="https://user-images.githubusercontent.com/5801015/195809916-477fd9c8-b1d6-4391-be60-0a88e0acac33.png">

**Comment by christian-reichart** [comment]
Didn't have this problem because I always have a "default" option which I set as defaultValue as well. Maybe this helps you out.

Otherwise users aren't able to reset the state after they have selected something.
